roles/keycloak/tasks/install_plugins.yml

```diff
@@ -0,0 +1,81 @@
+---
+
+### wayf plugin installation
+- name: Install wayf plugin
+  block:
+  - name: Create wayf plugin dir with parent directories
+    file:
+      path: "{{keycloak_home}}/modules/system/layers/keycloak/org/keycloak/{{ wayf.name }}/main"
+      state: directory
+
+  - name: Download wayf plugin jar
+    get_url:
+      url: "{{ wayf.jar_url }}"
+      dest: "{{keycloak_home}}/modules/system/layers/keycloak/org/keycloak/{{ wayf.name }}/main/"
+
+  - name: Install module.xml
+    get_url:
+      url: "{{ wayf.modulexml_url }}"
+      dest: "{{keycloak_home}}/modules/system/layers/keycloak/org/keycloak/{{ wayf.name }}/main/"
+
+  - name: Setup keycloak xml providers
+    xml:
```

Comment:
Can we have generic tasks that iterate over keycloak_plugins instead of having plugin-specific tasks?

Reply:
That's something i was also thinking about doing. I'll try to make it generic. Different types of plugin might require different/extra actions. But yes, i definitely agree that they should be somehow generic and iterate over the provided listing.

Reply:
Done, now plugins are loaded in a loop. Extra steps are now denoted by flags.